website/src/get-started/downloads.md

```diff
@@ -90,10 +90,15 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.20.0 (2020-03-01)
+Official [source code download](http://www.apache.org/dyn/closer.cgi/beam/2.20.0/apache-beam-2.20.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.20.0/apache-beam-2.20.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.20.0/apache-beam-2.20.0-source-release.zip.asc).
+
+[Release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12346582).
+
 ### 2.19.0 (2020-02-04)
-Official [source code download](http://www.apache.org/dyn/closer.cgi/beam/2.19.0/apache-beam-2.19.0-source-release.zip).
-[SHA-512](https://downloads.apache.org/beam/2.19.0/apache-beam-2.19.0-source-release.zip.sha512).
```

Comment:
Why do we want to delete these?

Reply:
I think it is the suggestion from https://beam.apache.org/contribute/release-guide/#deploy-source-release-to-distapacheorg?

Basically 2.19.0 is moved to `archive`

Reply:
Rolled it back.